client/src/components/profile/ProfileRequestForm.js

```diff
@@ -77,18 +156,21 @@ function ProfileRequestForm() {
               inputProps: { min: todayFormatted },
             }}
             type="date"
-            value={dropOffDate}
-            onChange={(e) => handleDropOffDateChange(e)}
+            value={startDate}
+            onChange={(e) => handleStartDateChange(e)}
             variant="outlined"
           ></TextField>
           <TextField
             select
             variant="outlined"
-            value={"9 am"}
+            value={startTime}
             style={{ width: "110px" }}
+            onChange={(e) => {
+              handleStartTimeChange(e);
+            }}
           >
-            <MenuItem value={"9 am"}>9 am</MenuItem>
-            <MenuItem value={"9:30 am"}>9:30 am</MenuItem>
+            <MenuItem value={"09:00:00"}>9 am</MenuItem>
```

Comment:
MUI should have a DateTime picker so you don't need to worry about handling this manually

Reply:
Changed to MUI Time Picker